.github/workflows/_e2e.yml

```diff
@@ -33,7 +34,7 @@ jobs:
   # Run e2e test against localnet built on the develop branch
   localnet:
     name: Localnet
-    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet || inputs.isRust || github.ref_name == 'develop'
+    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet || github.ref_name == 'develop'
```

Comment:
```suggestion
    if: inputs.isExplorer || inputs.isTypescriptSDK || inputs.isWallet
```
@DaughterOfMars should we also do this?

Reply:
Or would it prevent it from running from the nightly workflow?

Reply:
It would prevent it from running in nightly, yes

Reply:
But does leaving it means it will run on pushes to develop?

Reply:
I guess not because it won't be called by the hierarchy?

Reply:
Yes, exactly.